docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.        """
</pre><p> </p><p>Add the "Returns" section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>